### ISSUE TITLE
Failure to load plugin : Could not load file or assembly 'Microsoft.VisualStudio.Shell.11.0'

### ISSUE THREAD

**Issue by calumblair** [itechnology/FindUnusedFiles#12]
Using Visual Studdio 2013.
From the activity log:

``` xml
<entry>
    <record>632</record>
    <time>2015/10/14 13:04:27.855</time>
    <type>Information</type>
    <source>VisualStudio</source>
    <description>Begin package load [FindUnusedFilesPackage]</description>
    <guid>{0CF7B733-7D9B-45B9-9AA8-328AFD40580A}</guid>
  </entry>
  <entry>
    <record>633</record>
    <time>2015/10/14 13:04:27.867</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>CreateInstance failed for package [FindUnusedFilesPackage]</description>
    <guid>{0CF7B733-7D9B-45B9-9AA8-328AFD40580A}</guid>
    <hr>80070002</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.11.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
  <entry>
    <record>634</record>
    <time>2015/10/14 13:04:27.867</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [FindUnusedFilesPackage]</description>
    <guid>{0CF7B733-7D9B-45B9-9AA8-328AFD40580A}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.11.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
```

Some googling shows that changes such as this may help?
https://github.com/stbrowne/cppcheck-vs-addin/commit/597b957c876465eb79100ce683a30d954a06ee3d

**Comment by robert-hoffmann** [comment]
Strange, as you can see in the .csproj, there is no reference to Microsoft.VisualStudio.Shell.11.0 version 14 (14 is VS 2015)

https://github.com/itechnology/FindUnusedFiles/blob/master/FindUnusedFiles_VS/FindUnusedFiles_VS.csproj

Also it explicitly references:

```
<MinimumVisualStudioVersion>12.0</MinimumVisualStudioVersion>
<VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">11.0</VisualStudioVersion>
```

The reference to (11) is when it used to run under 2012 ..which it theoretically still should..

I however did build the project in 2015 for a moment, but switched back to 2013.
Try uninstalling the extension from VS2013, and then reinstalling it.

---

**Comment by robert-hoffmann** [comment]
Ok did some digging after recreating a new project in VS2013 (the project was originally created in 2012)

So after some diffing, i came up with this:
https://github.com/itechnology/FindUnusedFiles/tree/master/dist/archive

Try installing the 1.4.1 vsix and let me know if it works, if it does i'm gonna have to do some hacking to get it to work with 2012 too, but in the meantime you can just use this version :)

**Comment by calumblair** [comment]
OK. Just tried that and it only gave me the option to install it into VS2012 (which I hadn't realised I had  installed) and not 2013. Happy to close this issue as I've now removed the unused files  manually.

**Comment by robert-hoffmann** [comment]
Don't understand your last reply. Did it work finally ?

I'd like to fix this issue, but i have so much stuff on my pc everything just "works" ^^

**Comment by calumblair** [comment]
No, unfortunately  it didn't work but I have manually done what I had wanted to install this plugin to do.

**Comment by FatPanther** [comment]
@robert-hoffmann - I'd recommend reopening this - I'm having this issue (VS 2012).  Would love to be able to use this extension...

**Comment by robert-hoffmann** [comment]
Hi,

Sure, i would love to get this working again with older versions, especially since there is zero code making use of any VS version specific features. Heck, the majority of the code was written with VS 2012.

It's just VS apparently screws with your project files, or maybe the build part.

Anyways, can you give this build a try
[FindUnusedFiles-1.4.1.vsix](https://github.com/itechnology/FindUnusedFiles/blob/master/dist/archive/FindUnusedFiles-1.4.1.vsix)

If it doesn't work, i'd be happy to make a few builds with different options to try and get it working. ..i just need someone to try out the builds, since i don't have vs2012 on my PC anymore.

Not to mention, since i have 2013/2015 + SDKs installed, anything i build (even if i would reinstall 2012) would probably "just work" here since i have tons of dependency files flying around on my PC

**Comment by killergege** [comment]
I had the same issue, and tried the build you provided (1.4.1).
It seems to work !

**Comment by robert-hoffmann** [comment]
Cool thanks, maybe i'll upload that version to the marketplace !

**Comment by RichardPAsch** [comment]
Unable to get 1.4.1 version to work in VS2012 Enterprise edition.
Noticed you haven't updated this in 2 years - still an issue?
Tried install via VS ext. mgr UI and out-of process,, to no avail!
What is the status on this issue
